components/Sidebar/panels/Queue/index.tsx

```diff
@@ -36,7 +36,8 @@ const Queue = async () => {
             className="text-sm overflow-hidden text-ellipsis whitespace-nowrap"
           >
             <Link
-              href={`/app/track/${item.id}`}
+              // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+              href={item === null ? `` : `/app/track/${item.id}`}
```

Comment:
This will still throw an error because of `item.name` right? May be better to filter the queue array (inside the `getQueue` func) by item !== null